static/js/components/EducationDisplay.js

```diff
@@ -86,7 +86,7 @@ export default class EducationDisplay extends ProfileFormFields {
     }
     userPrivilegeCheck(profile, () => {
       rows.push(
-        <Cell col={12} className="profile-form-row" key={"I'm unique!"}>
+        <Cell col={12} className="profile-form-add-row" key={"I'm unique!"}>
```

Comment:
can you change this className to be `.profile-form-row.add` or `.profile-form-row.add-item`? you should then be able to apply the styling you want by doing:

```scss
.profile-form-row {
  // styles
  &.add {
    // styles
  }
}
```

instead of adding another class that receives the same styles as `.profile-form-row`